src/pycram/worlds/bullet_world.py

```diff
@@ -448,6 +449,9 @@ def __init__(self, world: World, mode: WorldMode):
         self.world = world
         self.mode: WorldMode = mode
 
+        if not "DISPLAY" in os.environ:
```

Comment:
document this. Otherwise some user with some weird display setup may complain about not having a bullet world and this will be hell to figure out why